index.html

```diff
@@ -232,47 +239,61 @@ <h4 id="about-intro">About me</h4>
       </section>
       <!-- contact section -->
       <footer id="contact-sections">
-        <div id="contact-intro">
-          <h4>
-            I'm always interested in hearing about new projects, so if you'd
-            like to chat please get in touch.
-          </h4>
-        </div>
-        <form
-          action="https://formspree.io/f/mbjwwwvr"
-          method="post"
-          id="personal-info"
-        >
-          <input
-            type="text"
-            placeholder="Full name"
-            id="fname"
-            name="fname"
-            maxlength="30"
-            required
-          />
-          <input
-            type="email"
-            id="email"
-            placeholder="Email address"
-            name="email"
-            required
-          />
-          <textarea
-            name="message"
-            id="message"
-            cols="30"
-            rows="10"
-            maxlength="500"
-            placeholder="Enter text here"
-            required
-          >
-          </textarea>
-          <div class="form-button">
-            <button type="submit">Get in touch</button>
+        <div class="form-control">
+          <div id="contact-intro">
+            <h4>
+              I'm always interested in hearing about new projects, so if you'd
+              like to chat please get in touch.
+            </h4>
           </div>
-        </form>
-        <ul class="socials">
+
+          <form
+            action="https://formspree.io/f/mbjwwwvr"
+            method="post"
+            id="personal-info"
+          >
+            <div class="names">
+              <input
+                type="text"
+                placeholder="Full name"
+                id="fname"
+                name="fname"
+                maxlength="30"
+                required
+              />
+              <input
+                type="text"
+                placeholder="Last name"
+                id="Last-name"
+                name="Last Name"
+                maxlength="30"
+                required
+                class="last-name"
+              />
+            </div>
+            <input
+              type="email"
+              id="email"
+              placeholder="Email address"
+              name="email"
+              required
+            />
+            <textarea
+              name="message"
+              id="message"
+              cols="30"
+              rows="10"
+              maxlength="500"
+              placeholder="Enter text here"
+              required
+            ></textarea>
+            <div class="form-button">
+              <button type="submit">Get in touch</button>
+            </div>
+          </form>
+        </div>
+
+        <ul class="socials2">
           <li>
             <a href="https://twitter.com/preateamie"
```

Comment:
- [ ] The social media icons should be displayed inline instead of blocked. Kindly use flex to display them horizontally just as it's in the Figma design. 

**See your mobile design view**
![image](https://user-images.githubusercontent.com/50941074/172000557-40cae519-85e8-47cb-a401-8e804e32d46a.png)

**Please match your design to this one**
- Please don't forget to center the icons on the page. 👍 

![image](https://user-images.githubusercontent.com/50941074/172000590-9d323430-7239-44da-af6f-10aa8a43e5ea.png)